spec/functional/resource/cron_spec.rb

```diff
@@ -63,8 +63,6 @@ def cron_should_not_exists(cron_name)
   end
 
   let(:new_resource) do
-    run_context = Chef::RunContext.new(Chef::Node.new, {}, Chef::EventDispatch::Dispatcher.new)
```

Comment:
The one with real platform information is defined directly above this and I forgot to remove the stub.